disperser/cmd/controller/flags/flags.go

```diff
@@ -183,6 +183,12 @@ var (
 		Required: false,
 		EnvVar:   common.PrefixEnvVar(envVarPrefix, "DETAILED_VALIDATOR_METRICS"),
 	}
+	EnablePerAccountBlobStatusMetricsFlag = cli.BoolTFlag{
+		Name:     common.PrefixFlag(FlagPrefix, "enable-per-account-blob-status-metrics"),
+		Usage:    "Whether to report per-account blob status metrics. If false, all metrics will be aggregated under account 0x0. (default: true)",
```

Comment:
What would be the alternative assuming we had a lot of Account IDs? I'm afraid that we would start relying on this metric for alerts and have to lose all the account labels in the future.

Reply:
> What would be the alternative assuming we had a lot of Account IDs?

Alternatively, we could define a set of keys that we care about, and only report those accounts directly. IMO it's a bit of a premature optimization to do that right now.

> start relying on this metric for alerts

I'm not sure it would make sense to configure alerts on a per-account basis. Do you have a reason in mind that we would want to do that?